test/extended/util/cli.go

```diff
@@ -148,6 +150,17 @@ func (c *CLI) SetupProject(name string, kubeClient *kclient.Client, _ map[string
 		e2e.Logf("Failed to create a project and namespace %q: %v", c.Namespace(), err)
 		return nil, err
 	}
+	if err := wait.ExponentialBackoff(kclient.DefaultBackoff, func() (bool, error) {
```

Comment:
Ick.  https://github.com/openshift/origin/blob/master/test/util/policy.go#L20.  Even a copy/paste would be preferable.